### ISSUE TITLE
libtexpdf error with Noto Color Emoji

### ISSUE THREAD

**Issue by McSinyx** [sile-typesetter/sile#1261]
I am having trouble with Noto Color Emoji, for example,

    \begin{document}
    \font[family = Noto Color Emoji]😄
    \end{document}

would give me

```console
$ sile emoji.sil 
SILE v0.12.0 (Lua 5.2)
<emoji.sil>
[1] 
libtexpdf:fatal: sfnt: table not found...
```

I am on NixOS unstable at the moment.  Other emoji fonts seem to work, including Noto Emoji (which is still the old blob one for some reason).

**Comment by alerque** [comment]
This *smells* to me like the problem is actually the font missing something it should have, but @simoncozens or @ctrlcctrlv can probably make a more educated guess at what is happening here. It's also possible we fumbles something, perhaps related to #1142/#1144.

**Comment by khaledhosny** [comment]
In luaotfload (for LuaTeX) I handle fonts like this by embedding the PNG returned by HarfBuzz directly in the PDF. Another option is to create a Type 3 font from the bitmaps (though I haven't tried this one myself).

**Comment by khaledhosny** [comment]
Note this is not a no standard font, the CBDT/CBLC is part of the OpenType spec. Another bitmap table with the same issue is sbix used by Apple emoji font.

**Comment by alerque** [comment]
I suppose if anybody out there chimes in again (@McSinyx?) to say this is something they would actually find useful we can re-open this issue to track adding CBDT/CBLC support. Personally I don't have the time or interest to code up support, but if somebody does come along that wants it enough to make it happen I'd be happy to review and facilitate a contribution.

**Comment by McSinyx** [comment]
I would find it useful, but it's not a high priority to me since I've switched back to the old (blob) Noto Emoji.  IMHO it'd be nice if this issue is reopened so maybe someone will be able to fix it in the future.  To quote [Drew DeVault](https://drewdevault.com/2021/10/26/stalebot.html):

> Let me offer you a different way of thinking about issues: a place for motivated users to collaborate on narrowing down the problem and planning a potential fix. A space for the community to work, rather than an action item for you to deal with personally. It gives people a place to record additional information, and, ultimately, put together a pull request for you to review. It does not matter if this process takes days or weeks or years to complete. Over time, the issue will accumulate details and workarounds to help users identify and diagnose the problem, and to provide information for the person that might eventually write a patch/pull request.

**Comment by ctrlcctrlv** [comment]
I agree with @khaledhosny, @alerque. This should be reopened and renamed to "CBDT/CBLC support". I also don't have the time or interest, but that's the right way to track it.

**Comment by ctrlcctrlv** [comment]
I just discovered that CBDT–CBLC and sbix can be made to share the same bitmap data in principle. I haven't made a Proof of Concept font yet but if this is the case, it means that if SILE supports one or the other it should be not so difficult to make cross-platform CBDT/sbix fonts out of any font that is only CBDT or only sbix. See https://twitter.com/fr_brennan/status/1478240370412638211 for the proposed method.
